test/runtests.jl

```diff
@@ -5,12 +5,13 @@ using Test, Profile
 # useful for testing
 stackframe(func, file, line; C=false) = StackFrame(Symbol(func), Symbol(file), line, nothing, C, false, 0)
 
+
 @testset "flamegraph" begin
     backtraces = UInt64[0, 4, 3, 2, 1,   # order: calles then caller
-                        0, 6, 5, 1,
```

Comment:
could we keep this whitespace? (Not a big deal either way.)

Reply:
For sure, atom messed up it for some reason.